src/sentry/options/defaults.py

```diff
@@ -10,15 +10,38 @@
 from sentry.options import (
     FLAG_IMMUTABLE, FLAG_NOSTORE, FLAG_PRIORITIZE_DISK, FLAG_REQUIRED, register
 )
-from sentry.utils.types import Dict, String
+from sentry.utils.types import Dict
 
+
+def default_from_email():
```

Comment:
iirc you cant generally just send email from a domain like this without getting caught in spam filters